WARNING: Unknown module: cpw.mods.bootstraplauncher specified to --add-exports
jlw: native file exists: 'C:\Users\Administrator\AppData\Roaming\PCL\libjlw-1.3.2.dll'.
jlw: native file checked.
2023-05-03 21:58:38,919 main WARN Advanced terminal features are not available in this environment
[21:58:38] [main/INFO] [cp.mo.mo.Launcher/MODLAUNCHER]: ModLauncher running: args [--username, anxiangge, --version, Random Stone-1.1.6, --gameDir, C:\Users\Administrator\Desktop\我的世界pcl版\.minecraft\versions\Random Stone-1.1.6, --assetsDir, C:\Users\Administrator\Desktop\我的世界pcl版\.minecraft\assets, --assetIndex, 1.16, --uuid, a643ee826d0d43cf8cefa6dc403be7db, --accessToken, ????????, --userType, msa, --versionType, PCL, --width, 854, --height, 480, --launchTarget, fmlclient, --fml.forgeVersion, 36.2.39, --fml.mcVersion, 1.16.5, --fml.forgeGroup, net.minecraftforge, --fml.mcpVersion, 20210115.111550]
[21:58:38] [main/INFO] [cp.mo.mo.Launcher/MODLAUNCHER]: ModLauncher 8.1.3+8.1.3+main-8.1.x.c94d18ec starting: java version 17.0.3 by Microsoft
[21:58:39] [main/INFO] [ne.mi.fm.lo.FixSSL/CORE]: Added Lets Encrypt root certificates as additional trust
[21:58:39] [main/INFO] [mixin/]: SpongePowered MIXIN Subsystem Version=0.8.4 Source=file:/C:/Users/Administrator/Desktop/我的世界pcl版/.minecraft/libraries/org/spongepowered/mixin/0.8.4/mixin-0.8.4.jar Service=ModLauncher Env=CLIENT
[21:58:39] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file Mod File: C:\Users\Administrator\Desktop\我的世界pcl版\.minecraft\versions\Random Stone-1.1.6\mods\辅助\I18nUpdateMod-3.4.2-all.jar is missing mods.toml file
[21:58:39] [main/WARN] [ne.mi.fm.lo.mo.ModDiscoverer/SCAN]: File C:\Users\Administrator\Desktop\我的世界pcl版\.minecraft\versions\Random Stone-1.1.6\mods\辅助\I18nUpdateMod-3.4.2-all.jar has been ignored - it is invalid
[21:58:40] [main/INFO] [STDERR/]: [org.antlr.v4.runtime.ConsoleErrorListener:syntaxError:38]: line 13:0 token recognition error at: '`'
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1077]: java.lang.reflect.InvocationTargetException
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1077]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1077]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1077]: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1077]: 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1077]: 	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:60)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1077]: 	at oolloo.jlw.Wrapper.main(Wrapper.java:53)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: Caused by: java.lang.ExceptionInInitializerError
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at org.openjdk.nashorn.internal.runtime.Context.compile(Context.java:1509)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at org.openjdk.nashorn.internal.runtime.Context.compileScript(Context.java:1449)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at org.openjdk.nashorn.internal.runtime.Context.compileScript(Context.java:759)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at org.openjdk.nashorn.api.scripting.NashornScriptEngine.compileImpl(NashornScriptEngine.java:528)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at org.openjdk.nashorn.api.scripting.NashornScriptEngine.compileImpl(NashornScriptEngine.java:517)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at org.openjdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:395)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at org.openjdk.nashorn.api.scripting.NashornScriptEngine.eval(NashornScriptEngine.java:146)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at java.scripting/javax.script.AbstractScriptEngine.eval(AbstractScriptEngine.java:247)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at net.minecraftforge.coremod.CoreMod.initialize(CoreMod.java:40)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at net.minecraftforge.coremod.CoreModEngine.initialize(CoreModEngine.java:75)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at net.minecraftforge.coremod.CoreModEngine.initializeCoreMods(CoreModEngine.java:69)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at net.minecraftforge.coremod.CoreModProvider.getCoreModTransformers(CoreModProvider.java:17)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at net.minecraftforge.fml.loading.FMLServiceProvider.transformers(FMLServiceProvider.java:142)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at cpw.mods.modlauncher.TransformationServiceDecorator.gatherTransformers(TransformationServiceDecorator.java:74)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at cpw.mods.modlauncher.TransformationServicesHandler.lambda$initialiseServiceTransformers$6(TransformationServicesHandler.java:101)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at java.base/java.util.HashMap$Values.forEach(HashMap.java:1065)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at cpw.mods.modlauncher.TransformationServicesHandler.initialiseServiceTransformers(TransformationServicesHandler.java:101)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at cpw.mods.modlauncher.TransformationServicesHandler.initializeTransformationServices(TransformationServicesHandler.java:64)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at cpw.mods.modlauncher.Launcher.run(Launcher.java:76)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	at cpw.mods.modlauncher.Launcher.main(Launcher.java:66)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1086]: 	... 6 more
[21:58:40] [main/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:659]: Caused by: java.lang.RuntimeException: java.lang.NoSuchMethodException: no such method: sun.misc.Unsafe.defineAnonymousClass(Class,byte[],Object[])Class/invokeVirtual
[21:58:40] [main/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:659]: 	at org.openjdk.nashorn.internal.runtime.Context$AnonymousContextCodeInstaller.lambda$getDefineAnonymousClass$0(Context.java:335)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:659]: 	at java.base/java.security.AccessController.doPrivileged(AccessController.java:318)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:659]: 	at org.openjdk.nashorn.internal.runtime.Context$AnonymousContextCodeInstaller.getDefineAnonymousClass(Context.java:327)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:659]: 	at org.openjdk.nashorn.internal.runtime.Context$AnonymousContextCodeInstaller.<clinit>(Context.java:317)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:659]: 	... 27 more
[21:58:40] [main/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:682]: Caused by: java.lang.NoSuchMethodException: no such method: sun.misc.Unsafe.defineAnonymousClass(Class,byte[],Object[])Class/invokeVirtual
[21:58:40] [main/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:682]: 	at java.base/java.lang.invoke.MemberName.makeAccessException(MemberName.java:976)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:682]: 	at java.base/java.lang.invoke.MemberName$Factory.resolveOrFail(MemberName.java:1117)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:682]: 	at java.base/java.lang.invoke.MethodHandles$Lookup.resolveOrFail(MethodHandles.java:3649)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:682]: 	at java.base/java.lang.invoke.MethodHandles$Lookup.findVirtual(MethodHandles.java:2680)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:682]: 	at org.openjdk.nashorn.internal.runtime.Context$AnonymousContextCodeInstaller.lambda$getDefineAnonymousClass$0(Context.java:329)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:682]: 	... 30 more
[21:58:40] [main/INFO] [STDERR/]: [java.lang.Throwable:printEnclosedStackTrace:724]: Caused by: java.lang.NoSuchMethodError: 'java.lang.Class sun.misc.Unsafe.defineAnonymousClass(java.lang.Class, byte[], java.lang.Object[])'
[21:58:40] [main/INFO] [STDERR/]: [java.lang.Throwable:printEnclosedStackTrace:724]: 	at java.base/java.lang.invoke.MethodHandleNatives.resolve(Native Method)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.Throwable:printEnclosedStackTrace:724]: 	at java.base/java.lang.invoke.MemberName$Factory.resolve(MemberName.java:1085)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.Throwable:printEnclosedStackTrace:724]: 	at java.base/java.lang.invoke.MemberName$Factory.resolveOrFail(MemberName.java:1114)
[21:58:40] [main/INFO] [STDERR/]: [java.lang.Throwable:printEnclosedStackTrace:724]: 	... 33 more
Exception in thread "main" 